.github/workflows/conformance.yml

```diff
@@ -13,13 +13,19 @@ jobs:
   everything:
     # host is irrelevant because everything will run in Docker containers
     runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: stable
           components: clippy, rustfmt
       - uses: extractions/setup-just@v3
+      - uses: docker/setup-buildx-action@v3
```

Comment:
Why is this useful/what does it fix? Maybe add 1-2 lines of comments on the rationale here?